Source/WebCore/page/SecurityOrigin.cpp

```diff
@@ -91,7 +91,7 @@ static RefPtr<SecurityOrigin> getCachedOrigin(const URL& url)
     return nullptr;
 }
 
-static bool shouldTreatAsOpaqueOrigin(const URL& url)
+bool SecurityOrigin::shouldTreatAsOpaqueOrigin(const URL& url)
```

Comment:
Okay, I'll move. I moved it previously, but then moved it back and left it in `SecurityOrigin` because:
 - it relies on the static function `schemeRequiresHost()`, so that will move, too
 - it's used in `SecurityOrigin::SecurityOrigin(const URL&)`, too
 - I was avoiding adding a dependency on `LegacySchemeRegistry` to `SecurityOriginData`
 
 but none of these prevent moving it to `SecurityOriginData`.